src/Sulu/Component/Rest/ListBuilder/Metadata/FieldDescriptorFactory.php

```diff
@@ -223,7 +228,7 @@ private function getCountFieldDescriptor(
             $propertyMetadata->getSearchability(),
             $propertyMetadata->getType(),
             $propertyMetadata->isSortable(),
-            $this->resolveOptions($propertyMetadata->getDistinct(), $options),
+            $propertyMetadata->getDistinct(),
```

Comment:
This code previously only worked because of two things:
* either the options array is empty then resolve options will return the value as is (and keep it as a boolean
* sulu doesn't use strict types and empty strings are false, and filled strings are true.